renderer/src/common/components/settings/tabs/version-tab.tsx

```diff
@@ -0,0 +1,62 @@
+import log from 'electron-log/renderer'
+import { Badge } from '../../ui/badge'
+import { useState, useEffect } from 'react'
+import { toast } from 'sonner'
+
+export function VersionTab() {
+  const [appInfo, setAppInfo] = useState<{
+    appVersion: string
+    isReleaseBuild: boolean
+    toolhiveVersion: string
+  }>({
+    appVersion: 'N/A',
+    isReleaseBuild: false,
+    toolhiveVersion: 'N/A',
+  })
+
+  useEffect(() => {
+    const fetchVersionInfo = async () => {
```

Comment:
I guess my only complaint is that I would move this to a separate function/hook to move it out of the component body (or even to a common file)

Reply:
it is really tiny and only fetching from ipc, but yep let me create a custom hook for this